samples/shell/src/main.c

```diff
@@ -37,11 +37,16 @@ void main(void)
 	npmx_ldsw_active_discharge_enable_set(npmx_ldsw_get(npmx_instance, 0), true);
 	npmx_ldsw_active_discharge_enable_set(npmx_ldsw_get(npmx_instance, 1), true);
 
-	/* Register callback so that POF warnings can be printed. */
-	npmx_pof_config_t pof_config = { .status = NPMX_POF_STATUS_ENABLE,
-					 .threshold = NPMX_POF_THRESHOLD_3V5,
-					 .polarity = NPMX_POF_POLARITY_HIGH };
+	npmx_pof_t *pof_instance = npmx_pof_get(npmx_instance, 0);
 
+	npmx_pof_config_t pof_config;
+	npmx_error_t err_code = npmx_pof_config_get(pof_instance, &pof_config);
+	if (err_code != NPMX_SUCCESS) {
+		LOG_INF("Error: unable to read POF config.");
```

Comment:
```suggestion
		LOG_ERR("Unable to read POF config.");
```
Line 29 could also be changed to error on this occasion.